tools/spring/spring-oauth2-utilities/src/main/java/org/opfab/springtools/configuration/oauth/OpfabAuthorizationManager.java

```diff
@@ -21,17 +21,15 @@ private OpfabAuthorizationManager() {
     }
 
     public static AuthorizationManager<RequestAuthorizationContext> authenticatedAndIpAllowed() {
-        return AuthorizationManagers.allOf(AuthenticatedAuthorizationManager.authenticated(),
-                            new OpfabIpAuthorizationManager());
+        return AuthorizationManagers.allOf(AuthenticatedAuthorizationManager.authenticated());
```

Comment:
Remove "ipAllowed" from methos names.
Not needed to call AuthorizationManagers.allOf(..) when there is just one AuthorizationManager

Reply:
Done.